docs/user/resources/02-logpipeline.md

```diff
@@ -199,18 +199,16 @@ The status of the LogPipeline is determined by the condition types `AgentHealthy
-| ConfigurationGenerated | False            | TLSCertificateKeyPairInvalid | TLS certificate and private key do not match                                                                                                                                                                                        |
+| Condition Type         | Condition Status | Condition Reason            | Condition Message                                                                                                                                                                                                                   |
+|------------------------|------------------|-----------------------------|-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| AgentHealthy           | True             | AgentReady              | Fluent Bit DaemonSet is ready                                                                                                                                                                                                       |
+| AgentHealthy           | False            | AgentNotReady           | Fluent Bit DaemonSet is not ready                                                                                                                                                                                                   |
+| ConfigurationGenerated | True             | ConfigurationGenerated      |                                                                                                                                                                                                                                     |
+| ConfigurationGenerated | False            | ReferencedSecretMissing     | One or more referenced Secrets are missing                                                                                                                                                                                          |
+| ConfigurationGenerated | False            | UnsupportedLokiOutput       | grafana-loki output is not supported anymore. For integration with a custom Loki installation, use the `custom` output and follow [Intergrate with Loki](https://kyma-project.io/#/telemetry-manager/user/integration/loki/README). |
+| ConfigurationGenerated | False            | TLSCertificateInvalid       | TLS certificate invalid                                                                                                                                                                                                             |
```

Comment:
In which reason you will cover "TLSCertificateKeyPairInvalid"?

Reply:
These two, together with "TLSCertificateInvalid" will be all covered in "TLSCertificateInvalid"